File: LojinhaManagerAPI/Models/Service.cs
namespace LojinhaManagerAPI.Models
{
    public class Service
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }

        public Service(int Id, string Name, string Description) 
        {
            this.Id = Id;
            this.Name = Name;
            this.Description = Description;
        }

    }
}

File: lojinha_manager_front_end/lib/app/models/service.dart
import 'package:flutter/material.dart';

class Service with ChangeNotifier {
  String description;
  bool isChecked;

  Service({
    required this.description,
    this.isChecked = false,
  });

  void changeSelection() {
    isChecked = !isChecked;
    notifyListeners();
  }
}
